sharness/t0052-service-state-export.sh

```diff
@@ -9,7 +9,8 @@ test_cluster_init
 
 
 test_expect_success IPFS,CLUSTER "state export fails without snapshots" '
-    cluster_kill && sleep 5 &&
+    cluster_kill
```

Comment:
Hmm why did getting rid of the &&s help?

Reply:
it didnt.. i think this is a leftover from trying stuff